### ISSUE TITLE
Implementation question for the calculation of p_emp

### ISSUE THREAD

**Issue by AlyssaYoung** [ShahryarBQ/DRO#1]
hi~I have a question about the calculation of p_emp. As you mentioned in Appendix B, you use exponential moving average to calculate the p_emp. Why in the calculation of p_batch, you implement it by `p_batch = torch.tensor([len(c) / len(train) for c in batch_by_class])` not `p_batch = torch.tensor([len(c) / len(X_batch) for c in batch_by_class])`? 

Moreover, can you further explain the motivation of EMA, i.e. when the exact value of empirical label distribution is not available?
